docs/guides/runners.md

```diff
@@ -0,0 +1,72 @@
+# Running benchmarks with the benchmark runner
+
+nnbench provides the `BenchmarkRunner` as a compact interface to collect and run benchmarks selectively.
+
+##  The abstract `BenchmarkRunner`  class
+Let's first instantiate and then walk through the abstract class.
```

Comment:
Maybe "base" instead of abstract? Abstract reminds me of `abc`, which we explicitly do not use.